.env-template

```diff
@@ -52,6 +52,9 @@ YAGNA_DATADIR="."
 ## ZkSync driver
 #ZKSYNC_RINKEBY_RPC_ADDRESS=https://rinkeby-api.zksync.io/jsrpc
 #ZKSYNC_MAINNET_RPC_ADDRESS=https://api.zksync.io/jsrpc
+#ZKSYNC_RINKEBY_GETH_ADDR=http://1.geth.testnet.golem.network:55555
+#ZKSYNC_MAINNET_GETH_ADDR=http://geth.golem.network:55555
```

Comment:
Not sure if we should have separate env variables here. Are there legit cases when we would use different geth nodes for different drivers?

Reply:
It was suggested by @maaktweluit 

Even when we already have `ERC20_MAINNET_GETH_ADDRESS` and the `RINKEBY` version,
they are placed under __erc20__ driver section, so I think we can either: 
- prepare token unspecific geth node addresses (supporting all cases) 
- leave it as is.

WDYT?

Reply:
I would rename he variables and use them in both drivers.
`ERC20_RINKEBY_GETH_ADDR` -> `RINKEBY_GETH_ADDR`
`ERC20_MAINNET_GETH_ADDR` -> `MAINNET_GETH_ADDR`